src/app/admin/modules/data-management/components/data-graph-child/data-graph-child.component.ts

```diff
@@ -0,0 +1,245 @@
+import { Component, OnInit , Input, ViewChild, ElementRef} from '@angular/core';
+import * as echarts from 'echarts';
+import { EChartOption } from 'echarts';
+import { Graph } from 'src/app/shared/interfaces/graph';
+import { GraphData } from 'src/app/shared/interfaces/graph-data';
+import { PieGraphData } from 'src/app/shared/interfaces/pie-graph-data';
+
+@Component({
+  selector: 'app-data-graph-child',
+  templateUrl: './data-graph-child.component.html',
+  styleUrls: ['./data-graph-child.component.css']
+})
+
+
+export class DataGraphChildComponent implements OnInit {
+  @ViewChild('myCharts') myCharts: ElementRef;
+
+  @Input() titleGraphNames?: Graph[];
+  @Input() isPieGraph: boolean;
+
+  private titleYear: string;
+  private titleDepartment: string;
+  private pieGraphData?: PieGraphData[] = [];
+
+  chartOption?: EChartOption;
+  title: string;
+  xAxisList: string[] = ['a', 'b', 'c', 'd', 'e'];
+  seriesData: GraphData[] = [{id: 0, data: [120, 101, 90, 134, 230, 132, 210]}, {id: 1, data: [320, 301, 390, 302, 330, 320, 334]}];
```

Comment:
For mocked data, please add comments for them or name these variable explicitly.